integreat_cms/cms/templates/translations/translations_management.html

```diff
@@ -0,0 +1,43 @@
+{% extends "_base.html" %}
+{% load l10n %}
+{% load i18n %}
+{% load static %}
+{% load humanize %}
+{% load widget_tweaks %}
+{% block content %}
+    <form enctype="multipart/form-data" method="post" data-unsaved-warning>
+        {% csrf_token %}
+        <div class="flex flex-wrap justify-between mb-4">
+            <h1 class="heading overflow-hidden text-ellipsis">{% translate "Manage Translations" %}</h1>
+            {% if perms.cms.manage_translations %}
+                <button class="btn ml-auto mr-0">{% translate "Save" %}</button>
+            {% endif %}
+        </div>
+        <div class="grid xl:grid-cols-2 2xl:grid-cols-3 gap-4">
+            <div class="rounded border border-solid border-blue-500 shadow-2xl bg-white">
+                <div class="p-4 rounded bg-water-500">
+                    <h3 class="heading font-bold text-black">
+                        <i icon-name="bot" class="mr-2"></i> {% translate "Machine Translation" %}
+                    </h3>
+                </div>
+                <div class="px-4 pb-4 divide-y divide-gray-200 space-y-4">
+                    <div>
+                        <!-- Options related to the type of content which can be machine translated -->
+                        <label>{% trans "Automatic translations" %}</label>
+                        <div class="help-text">{% translate "Automatically machine translate the following types of content:" %}</div>
+                        <label for="{{ form.machine_translate_pages.id_for_label }}">{{ form.machine_translate_pages.label }}</label>
+                        {% render_field form.machine_translate_pages %}
+                        <label for="{{ form.machine_translate_events.id_for_label }}">{{ form.machine_translate_events.label }}</label>
+                        {% render_field form.machine_translate_events %}
+                        <label for="{{ form.machine_translate_pois.id_for_label }}">{{ form.machine_translate_pois.label }}</label>
+                        {% render_field form.machine_translate_pois %}
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+</form>
+{{ media_config_data|json_script:"media_config_data" }}
```

Comment:
```suggestion
```
There will probably not be a media library on this page, so we can remove the corresponding config data